packages/cli/src/commands/init.ts

```diff
@@ -100,7 +100,10 @@ async function handleOnboardingFlow(config: ConfigService) {
     spinner = ora('Setting up your new account').start();
 
     await createOrganizationHandler(config, answers);
-    const applicationIdentifier = await createEnvironmentHandler(config, answers);
+    const existingEnvironment = await getEnvironmentMe();
```

Comment:
I am not familiar with the task, we do not need to check if the environment already exists, and if it's not to create one? because it was created on the flow before?

Reply:
The environments are created during the creation of the organization in this case, so we never create environments manually as before.

Reply:
I see. 
🚀